client/devdocs/style.scss

```diff
@@ -155,7 +155,6 @@ $devdocs-max-width: 720px;
 		// A wrapper inside a wrapper to help reliably center
 		// component and block examples.
 		.docs-example__wrapper-content-centering {
-			display: table;
```

Comment:
Why is this change necessary? It doesn't seem related to Dot Pager or Swipeable

Reply:
The reason why it is necessary is that the dot pages component of the dev docs doesn't work as expected when `display: table` is set here.

Reply:
Got it. We should make sure that this change doesn't introduce a regression for other components that were potentially relying on this CSS rule for their docs.

Reply:
I tested 
[/devdocs/design](http://calypso.localhost:3000/devdocs/design) and 
[/devdocs/blocks](http://calypso.localhost:3000/devdocs/blocks) and I wasn't able to notice anything broken.